### PR TITLE
Add syntax highlighting

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

File: user/languages/android.md [hunk 1]
```diff
@@ -179,9 +179,10 @@ This can be overridden as described in the [general build configuration](/user/c
 
 A peculiarity of dependency caching in Gradle means that to avoid uploading the cache after every build you need to add the following lines to your `.travis.yml`:
 
-```
+```yaml
 before_cache:
-  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
```
Comment (SUPERCILEX) [inline user/languages/android.md]: Copied from Java which should be the same: https://docs.travis-ci.com/user/languages/java/#Caching.

**Comment by BanzaiMan** [comment]
@plaindocs Could you review this? Thank you! <3

**Comment by plaindocs** [comment]
Sure thing! :heart:

**Comment by plaindocs** [comment]
And done. Thank you so much!

**Comment by SUPERCILEX** [comment]
Thanks!

**Pull request merged**